course3/UniquePtr/ptr_test.cc

```diff
@@ -0,0 +1,73 @@
+#include "my_unique_ptr.h"
+#include <array>
+#include <string>
+#include <cassert>
+#include <iostream>
+#include <cstring>
+#include <memory>
+
+
+struct Foo
+{
+    Foo()      { std::cout << "Foo::Foo\n";  }
+    ~Foo()     { std::cout << "Foo::~Foo\n"; }
+    void bar() { std::cout << "Foo::bar\n";  }
```

Comment:
Neither of this methods actually touches the object's data, they all return the same constants. As such, they are useless in testing the object's identity.